stellar/deploy-contract.js

```diff
@@ -59,9 +61,15 @@ function getWasmUrl(contractName, version) {
         throw new Error(`Unsupported contract ${contractName} for versioned deployment`);
     }
 
-    const pathName = contractName.replace(/_/g, '-');
+    // Convert `AxelarGateway` → `stellar-axelar-gateway`
+    const dirPath = `stellar-${contractName
```

Comment:
create a helper `pascalToKebab` and use that, the stellar prefix can be added after